ibm_mq/tests/test_utils.py

```diff
@@ -21,3 +22,29 @@ def test_sanitize_strings(input_string, expected):
     from datadog_checks.ibm_mq.utils import sanitize_strings
 
     assert expected == sanitize_strings(input_string)
+
+
+@pytest.mark.parametrize(
+    'datestamp,timestamp',
+    [
+        pytest.param('2021-09-08', '19.19.41', id='elapsed a'),
+        pytest.param('2020-01-01', '10.25.13', id='elapsed b'),
+        pytest.param('2021-08-01', '12.00.00', id='elapsed c'),
+    ],
+)
+@pytest.mark.skipif(PY2, reason='Test only available on Python 3')
+def test_calculate_elapsed_time(datestamp, timestamp):
+    import time
+    from datetime import datetime
+
+    from datadog_checks.ibm_mq.utils import calculate_elapsed_time
+
+    current_tz = time.tzname[time.daylight]
```

Comment:
Related to https://github.com/DataDog/integrations-core/pull/10129/files#r709524727, should we test with an actual current timestamp and randomly generated timestamps instead?